css/styles/style.css

```diff
@@ -0,0 +1,80 @@
+body {
+    font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+    max-width: 900px;
+    margin: 0 auto;
+    padding: 3rem;
+}
+
+/* contributors section styles*/
+
+@media screen and (max-width: 400px) {
```

Comment:
It  best practice is to add media queries at the end of the CSS file. and i suggest implementing best practices if it is possible Alex.